docs/decision_records/api/API-010-appcallback-versioning.md

```diff
@@ -0,0 +1,14 @@
+# API-010: App Callback Versioning
+
+## Status
+Rejected

```

Comment:
This will be `Accepted` Status because we accept to not version http appcallback :)

Reply:
fixed